cmake/projects/libsodium/hunter.cmake

```diff
@@ -17,10 +18,14 @@ hunter_add_version(
     00a999629f1f8f500cef29818135f8f889f44e41
 )
 
+hunter_configuration_types(libsodium CONFIGURATION_TYPES Release)
 if(MSVC)
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_msbuild)
 else()
     hunter_pick_scheme(DEFAULT url_sha1_libsodium_autogen_autotools)
```

Comment:
> want me to change it to include this?

@Cyberunner23 Please take a look at this pull request I think you just need to use newly introduced feature.

@pretyman Okay, since this affect only one package with separate build scheme, let's ignore it for now.